src/Components/Artwork/Details.tsx

```diff
@@ -139,7 +139,7 @@ const HighDemandInfo = () => {
   return (
     <Flex flexDirection="row" alignItems="center">
       <HighDemandIcon width={16} height={16} />
-      <Text variant="xs" color="blue100" ml={0.3}>
+      <Text variant="xs" color="blue100" ml="3px">
```

Comment:
I guess but it sounds like you just want a normal `" "` or something

Reply:
Hmm, placing a `{" "}` behind`<HighDemandIcon width={16} height={16} />` or inside of `Text`  doesn't seem to work 🤔

Reply:
Inside of `Text` `&nbsp;` would work

Reply:
It works 👍